frontend/main-app/src/components/Login/LoginUtils.ts

```diff
@@ -37,4 +37,12 @@ export const axiosPostForm = async (
     console.log(error);
   }
   return false;
-}
+};
+
+export const getApiUrl = (): string => {
+  if (process.env.NODE_ENV === "development") {
+    return process.env.API_URL_DEVELOPMENT;
+  } else {
```

Comment:
Maybe we need to check also if they are not defined. like this:
` else if (process.env.API_URL_PRODUCTION) {
    return process.env.API_URL_PRODUCTION;
  } else `{`
    throw new Error("API URLs are not defined");
  }`

Reply:
Good catch!

In the latest commit, I added error handling for three cases:
* API_URL_DEVELOPMENT is not set in the .env file.
* API_URL_PRODUCTION is not set in the .env file.
* Development/production mode is not set in the package.json startup script.